src/components/menus/Pokemon.ts

```diff
@@ -208,6 +208,26 @@ export class Pokemon<TGameStartr extends FullScreenPokemon> extends GeneralCompo
         this.gameStarter.menuGrapher.setActiveMenu("PokemonMenuContext");
     }
 
+    /**
+     * Opens up the status menu for a Pokemon.
+     *
+     * @param pokemon   A Pokemon to show statistics of.
+     */
+    private addStatus(pokemon: IPokemon) {
+        const status = pokemon.status;
+        if (status === "frozen") {
+            this.gameStarter.menuGrapher.addMenuDialog("PokemonMenuStatsStatus", "FRZ");
+        } else if (status === "paralyzed") {
+            this.gameStarter.menuGrapher.addMenuDialog("PokemonMenuStatsStatus", "PAR");
+        } else if (status === "poison") {
+            this.gameStarter.menuGrapher.addMenuDialog("PokemonMenuStatsStatus", "PSN");
+        } else if (status === "sleep") {
+            this.gameStarter.menuGrapher.addMenuDialog("PokemonMenuStatsStatus", "SLP");
+        } else {
+            this.gameStarter.menuGrapher.addMenuDialog("PokemonMenuStatsStatus", "OK");
```

Comment:
This method calls `this.gameStarter.menuGrapher.addMenuDialog("PokemonMenuStatsStatus", ` in each case. Instead, make it return the new status string, and have its caller call all that. Less code overall.